cmd/get.go

```diff
@@ -80,25 +116,42 @@ func initGet(kinds schema.KindCatalog) {
 					parentValue[i] = *v
 				}
 				var err error
+
+				format, err := cmd.Flags().GetString("output")
+				if err != nil {
+					fmt.Println("Error retrieving output format:", err)
+					return
+				}
+
+				if format != "json" && format != "yaml" && format != "name" {
+					fmt.Fprintf(os.Stderr, "Invalid output format: %s. Expected 'json' or 'yaml' or 'name'.\n", format)
+					return
+				}
```

Comment:
You could use: https://pkg.go.dev/github.com/thediveo/enumflag
Hopefully it will make autocompletion better

Reply:
thanks, will do!